sechub-scan/src/main/java/com/daimler/sechub/domain/scan/product/ProductExecutorCallbackImpl.java

```diff
@@ -66,7 +67,7 @@ public AdapterMetaDataConverter getMetaDataConverter() {
     @Override
     public ProductResult save(ProductResult result) {
         if (result == null) {
-            LOG.error("Product executor {} returned null as one of the results {}", productIdentifier, context.getTraceLogId());
+            LOG.error("Product executor {} returned null as one of the results {}", productExecutorContext.getExecutorConfig(), context.getTraceLogId());
```

Comment:
At this point it can only be ONE result. But the `ProductExecutor.execute(..)` method can return multiple results. Why? Because some products  must be called multiple times . For example: When we scan more than one URL with Netsparker we must scan each URL extra, because REST API does not allow multiple instances. So the executor will iterate over all URLs and return a list of all product results.

In this case we will have multiple scan log entries  ala 

```
NETSPARKER_${configUUID}.json
NETSPARKER_${configUUID}[1].json
NETSPARKER_${configUUID}[2].json
```

To avoid confusion I will change the comment as proposed.